tools/obisindicators/obisindicators.xml

```diff
@@ -53,7 +53,7 @@
             <option value="0">Robinson projection</option>
             <option value="4326">Geographic projection</option>
         </param>
-        <param name="resolution" label="Choose a resolution for the discrete global grid specifying a number between 1 and 30" type="text" data_ref="input" help="We strongly advise to stay below 15 or else you won't see a thing. The bigger your dataset is the smaller the resolution should be."/>
+        <param name="resolution" label="Choose a resolution for the discrete global grid (a number between 1 and 30)" type="text" data_ref="input" help="We strongly advise to stay below 15 or else you won't see a thing. The bigger your dataset is the smaller the resolution should be."/>
```

Comment:
wait, why is this not an integer with min/max?

Reply:
You're right! Done!